### PR TITLE
Staff Part

### PR DESCRIPTION
**Pull request by DmitryPateew**
Add loading data in the table
Add loading data in staff card about candidates

### CODE REVIEW

File: src/components/candidates/candidate_list/candidates_list.tsx [hunk 1]
```diff
@@ -37,10 +38,10 @@ const CandidatesList: React.FC = () => {
 
     candidateService
       .getAllCandidates(params)
-      .then(({ data }) => {
+      .then(( data:any ) => {
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_list/candidates_list.tsx:41]: `(data: TCandidate[])`

File: src/components/common/checkbox/checkbox.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const areEqual = (prevProps: any, nextProps: any) => {
 //   const classes = useStyles();
 
 //   return (
-//     <Checkbox
+//     <checkbox
```
Comment (ArteGon) [inline src/components/common/checkbox/checkbox.tsx:30]: Custom checkbox?

File: src/components/staff/candidateTrello/api.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  baseURL: 'http://localhost:8085/api',
+});
+
+export const getCandidateIdByEmpolee=(id:number)=> {
+  return instance.get(`interviewtime?search=empId==${id}`).then(response => {
+    return response.data;
+  });
+}
+
+export const getCandidateById=(candidateIds:any[])=> {
```
Comment (ArteGon) [inline src/components/staff/candidateTrello/api.ts:13]: Can you change type any into other type?

File: src/components/candidates/candidate_list/candidates_list.tsx [hunk 4]
```diff
@@ -37,10 +38,10 @@ const CandidatesList: React.FC = () => {
...
         setCandidatesList(data);
       })
-      .catch(err => {
+      .catch((err:any) => {
         console.log(err);
```
Comment (ArteGon) [inline src/components/candidates/candidate_list/candidates_list.tsx:45]: delete console.log, if you don't need

File: src/components/staff/candidateTrello/candidateTrello.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+      ["id"]: item.cnId, ["interviewDate"]: item.beginDate
+    });
+  });
+
+  candidatetsData.forEach((item, index) => {
+    mass[index]["firstName"] = candidatetsData[index].firstName;
+    mass[index]["lastName"] = candidatetsData[index].lastName;
+    mass[index]["primaryTechnology"] = candidatetsData[index].mainSkill;
+  });
+  console.log(candidatetsData);
```
Comment (ArteGon) [inline src/components/staff/candidateTrello/candidateTrello.tsx]: delete console.log, if you don't need

File: src/components/staff/staffPage/staffPage.tsx [hunk 6]
```diff
@@ -61,7 +62,6 @@ const StaffPage: React.FC = () => {
   useEffect(() => {
     createUrl.add === 'add' && setAddMode(true);
   }, []);
-  console.log(createUrl.add);
```
Comment (ArteGon) [inline src/components/staff/staffPage/staffPage.tsx:64]: delete console.log, if you don't need

File: src/components/staff/tsTable/tsTable.tsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+          },
+          {
+            Header: 'Delete',
+            accessor: '',
+            disableFilters: true,
+            Cell: ({ cell }: any) => (
+              <button
+                value={cell.row.id}
+                onClick={() => {
+                  console.log(cell.row.original.id);
```
Comment (ArteGon) [inline src/components/staff/tsTable/tsTable.tsx:68]: delete console.log, if you don't need

**Review by ArteGon — DISMISSED** [review]

**Review by vchepurnaya — DISMISSED** [review]

**Review by ZmicerBoksha — APPROVED** [review]

**Review by vchepurnaya — APPROVED** [review]

**Review by ArteGon — APPROVED** [review]

**Pull request merged**